### ISSUE TITLE
[Accessibility issue] Vignettes lacks alt= in plots

### ISSUE THREAD

**Issue by MichaelChirico** [juliasilge/tidytext#231]
We have a tool that flags images that lack an `alt=` attribute. See https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices -- these are essential for conveying the image contents to users on screen readers.

You can add these to your vignettes by using the recent-ish `fig.alt` chunk option in `knitr`; see https://www.rstudio.com/blog/knitr-fig-alt/.

Unfortunately the lines are flagged in the HTML output, which won't correspond exactly to the .Rmd file lines, but I assume if you're familiar with the vignette it will be clear which images need it:

```
tidytext/inst/doc/troubleshooting.html:490
```
